packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -128,7 +128,17 @@ class HotRunner extends ResidentRunner {
       printError('Error initializing DevFS: $error');
       return 3;
     }
+    Stopwatch initialUpdateDevFSsTimer;
+    if (benchmarkMode) {
+      initialUpdateDevFSsTimer = new Stopwatch();
+      initialUpdateDevFSsTimer.start();
+    }
     final bool devfsResult = await _updateDevFS();
+    if (benchmarkMode) {
```

Comment:
I know you did not start this trend, but I'm not sure why we're recording benchmark data conditionally. It costs nothing. We should probably only consider `benchmarkMode` when outputting this data. I think removing some of these conditionals could simplify our code.